### ISSUE TITLE
Option to disable gym ticks in dashboard

### ISSUE THREAD

**Issue by quaestor** [theCrag/website#3366]
Feature request: add a checkbox in the "Streams" page in the settings for gyms.

**What you expected:**
![screenshot 2019-02-15 16 57 35](https://user-images.githubusercontent.com/54371/52868168-dc970c80-3142-11e9-9841-4f9a6ad7a777.png)

**Comment by rouletout** [comment]
+1 - would like to see that too

**Comment by scd** [comment]
Yes we missed this a while ago. I agree that this should be an option.

**Comment by quaestor** [comment]
Any news on this?

**Comment by birgander2** [comment]
+1
